src/ui/rideWindow.ts

```diff
@@ -21,10 +22,10 @@ export const model = new RideViewModel();
  * Window to edit properties of the currently selected ride.
  */
 export const rideWindow = window({
-	title: model._title,
+	title: "Edit ride",
```

Comment:
Does it not work with the ride's title in there?

Reply:
Reverted it back. Something didn't work out when I was trying out stuff and forget to revert it to the original.